android/app/src/main/java/net/minetest/minetest/UnzipService.java

```diff
@@ -210,7 +216,9 @@ private void migrate(Notification.Builder notificationBuilder, File newLocation)
 			return;
 
 		publishProgress(notificationBuilder, R.string.migrating, 0);
-		newLocation.mkdir();
+		if (!newLocation.mkdir()) {
```

Comment:
if it failed, why we should continue ? if the failure cannot permit to continue code we will just trigger side effects

Reply:
(see [my comment](https://github.com/minetest/minetest/pull/12960#issuecomment-1327972518))